README.md

```diff
@@ -19,14 +19,17 @@ We also welcome rule contributions directly to this repository! Since this repo
 
 If you have more questions, please see the [FAQ section in the Semgrep docs](https://semgrep.dev/docs/faq).
 
-## Security Coverage
+## Running Rules in CI/Pre-Commit/Developer Workflow
 
-`semgrep` features security rules that target [common weaknesses](https://cwe.mitre.org/) and [OWASP categories](https://owasp.org/www-project-top-ten/). Each `security` rule in this repository has metadata fields for `cwe` (and `owasp` when applicable). OWASP coverage for rules in this repository, organized by language, is shown below.
+If you want run these rules rather than write them, [see the CI instructions on the Semgrep Registry](https://semgrep.dev/explore) (click through to any rule pack). That website is a convenient frontend for this repository.
```

Comment:
@ievans Rather than casting the Registry as a convenient frontend to this repo, could we instead link to specific rulesets containing these rules? Going a little further, could we explain that we publish these rules to the Registry for convenient across-project distribution, as do others. Maybe we could link to Awesome Rulesets instructions.

Reply:
Did a major rewrite -- take another look!